src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -160,6 +160,8 @@ class SubjectHeading extends React.Component {
         search: '',
       },
       container,
+      seeMoreText,
+      seeMoreLinkUrl,
```

Comment:
Why do these need to be props that end up only being used as props to pass to another component?

Reply:
See above. If the props are not passed (barring some refactoring to make the SubjectHeadingsTable and SubjectHeadings not render each other), we need to use a conditional. I agree the chain of prop-passing is a drawback of this approach.